tests/network/ping.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: iputils
+# Summary: Ping another machine
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use utils;
+use lockapi;
+use mmapi 'wait_for_children';
+
+sub run {
+    if (get_var('IS_MM_SERVER')) {
+        barrier_create 'SETUP_DONE', 2;
+        mutex_create 'barrier_setup_done';
+    }
+
+    mutex_wait 'barrier_setup_done';
+
+    select_serial_terminal;
+    barrier_wait 'SETUP_DONE';
+
+    if (get_var('IS_MM_CLIENT')) {
+        my $remote = '10.0.2.101';
+        my $size = get_var('MM_MTU', 1380) - 28;    # 8 bytes ICMP and 20 bytes IPv4 header
+        script_retry("ping -M do -s $size -c1 -W1 $remote", retry => 5, delay => 2, fail_message => "Failed to ping $remote");
```

Comment:
This is duplicating `utils::ping_size_check`

Reply:
No it is not.
`ping_size_check` does `assert_script_run´ while I do `script_retry` to make this test more robust.

Reply:
then backport the changes, taking into account the behavior that ping_size_check does.

Reply:
done.